lesson_9/mosut_sorin/hw_lesson9_classes_mosut_sorin.py

```diff
@@ -0,0 +1,149 @@
+
+
+'''
+Build two classes of your choice that can model a real-life example.
+The class needs to meet the following requirements:
+
+    at least 5 attributes each
+
+    at least 2 methods each
+
+    one class to inherit from another
+
+As a demonstration create at least 5 instances of
+one class (preferably the child class) and
+call all the methods it holds
+
+Ex: You can have one class (Country) that has general attributes
+about countries such as area, neighbours, cities etc and
+methods related to those attributes. The second class can be a
+specific country (Romania) that has more specific attributes
+such as attractions, universities etc.
+'''
+
+# main class (parent class)
+class Automobil(object):
+    clasa = 'automobil'
+
+    def __init__(self, marca, tip, varianta, an, cilindree, putere):
+        self.marca = marca
+        self.tip = tip
+        self.var = varianta
+        self.an = an
+        self.cil = int(cilindree)
+        self.putere = int(putere)
+
+# method that converts the power from [kW] in [hp] and compare it wit a specific value
+    def test_putere(self):
+        if int(self.putere * 1.36) >= 200:
+            print(f'\nAutomobilul {self.marca} {self.var} din {self.an} dezvolta',
+                  f'{int(self.putere * 1.36)} cai putere, automobil sportiv.')
+        else:
+            print(f'\nAutomobilul {self.marca} {self.var} din {self.an} dezvolta',
+                  f'{int(self.putere * 1.36)} cai putere, are putere medie.')
+
+# method that print a message
+    @staticmethod
+    def print_step():
+        print('\nTEST DE PUTERE')
+
+# method that calculate the taxes for a vahicle
+    def impozit(self):
+        for x in range(len(test_cilindree) - 1):
```

Comment:
`test_cilindree` is a global variable. You can set it as a class attribute since global variables are not encouraged

Reply:
Also, please replace the iteration with range and len with: `for x in test_cilindree` if you do this you can use `x` instead of `test_cilindree[x] `

Reply:
I have use this type of iteration becouse i need in the same moment 2 consecutive values from list an by call [x] and [x+1] i could obtain them.